[2025-01-01T04:10:26.436+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-01-01T04:10:26.474+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: scheduler_dag_with_cron_exp_v1.my_name scheduled__2024-12-25T00:00:00+00:00 [queued]>
[2025-01-01T04:10:26.489+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: scheduler_dag_with_cron_exp_v1.my_name scheduled__2024-12-25T00:00:00+00:00 [queued]>
[2025-01-01T04:10:26.489+0000] {taskinstance.py:2866} INFO - Starting attempt 2 of 7
[2025-01-01T04:10:26.507+0000] {taskinstance.py:2889} INFO - Executing <Task(BashOperator): my_name> on 2024-12-25 00:00:00+00:00
[2025-01-01T04:10:26.515+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=114) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-01-01T04:10:26.517+0000] {standard_task_runner.py:72} INFO - Started process 119 to run task
[2025-01-01T04:10:26.517+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'scheduler_dag_with_cron_exp_v1', 'my_name', 'scheduled__2024-12-25T00:00:00+00:00', '--job-id', '94', '--raw', '--subdir', 'DAGS_FOLDER/scheduler_dag_with_cron_exp.py', '--cfg-path', '/tmp/tmplmif4ksz']
[2025-01-01T04:10:26.519+0000] {standard_task_runner.py:105} INFO - Job 94: Subtask my_name
[2025-01-01T04:10:26.583+0000] {task_command.py:467} INFO - Running <TaskInstance: scheduler_dag_with_cron_exp_v1.my_name scheduled__2024-12-25T00:00:00+00:00 [running]> on host b0ef9544b3b5
[2025-01-01T04:10:27.132+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='tanzid' AIRFLOW_CTX_DAG_ID='scheduler_dag_with_cron_exp_v1' AIRFLOW_CTX_TASK_ID='my_name' AIRFLOW_CTX_EXECUTION_DATE='2024-12-25T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='2' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-12-25T00:00:00+00:00'
[2025-01-01T04:10:27.133+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-01-01T04:10:27.172+0000] {subprocess.py:78} INFO - Tmp dir root location: /tmp
[2025-01-01T04:10:27.177+0000] {subprocess.py:88} INFO - Running command: ['/usr/bin/bash', '-c', 'echo My Name is Md. Tanzid Hasan']
[2025-01-01T04:10:27.199+0000] {subprocess.py:99} INFO - Output:
[2025-01-01T04:10:27.200+0000] {subprocess.py:106} INFO - My Name is Md. Tanzid Hasan
[2025-01-01T04:10:27.201+0000] {subprocess.py:110} INFO - Command exited with return code 0
[2025-01-01T04:10:27.279+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-01-01T04:10:27.280+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=scheduler_dag_with_cron_exp_v1, task_id=my_name, run_id=scheduled__2024-12-25T00:00:00+00:00, execution_date=20241225T000000, start_date=20250101T041026, end_date=20250101T041027
[2025-01-01T04:10:27.320+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-01-01T04:10:27.343+0000] {taskinstance.py:3895} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-01-01T04:10:27.354+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
